test/unit/jest.config.js

```diff
@@ -45,6 +45,6 @@ module.exports = {
 	],
 	reporters: [
 		'default',
-		'<rootDir>packages/scripts/config/jest-github-actions-reporter.js',
+		'<rootDir>packages/scripts/config/jest-github-actions-reporter/index.js',
```

Comment:
I moved this reporter to a subfolder to ensure we have configs only at the root level of the `config` directory.